modules/apps/27-interchain-accounts/host/ibc_module_test.go

```diff
@@ -337,6 +458,114 @@ func (suite *InterchainAccountsTestSuite) TestOnRecvPacket() {
 
 }
 
+func (suite *InterchainAccountsTestSuite) TestOnAcknowledgementPacket() {
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"ICA OnAcknowledgementPacket fails with ErrInvalidChannelFlow", func() {}, false,
+		},
+	}
+
+	for _, tc := range testCases {
+		tc := tc
+
+		suite.Run(tc.name, func() {
+			suite.SetupTest() // reset
+
+			path := NewICAPath(suite.chainA, suite.chainB)
+			suite.coordinator.SetupConnections(path)
+
+			err := SetupICAPath(path, TestOwnerAddress)
+			suite.Require().NoError(err)
+
+			tc.malleate() // malleate mutates test data
+
+			module, _, err := suite.chainA.App.GetIBCKeeper().PortKeeper.LookupModuleByPort(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID)
```

Comment:
I think this fixes the failing test

```suggestion
			module, _, err := suite.chainA.App.GetIBCKeeper().PortKeeper.LookupModuleByPort(suite.chainA.GetContext(), path.EndpointB.ChannelConfig.PortID)
```